modules/SofaGraphComponent/RequiredPlugin.cpp

```diff
@@ -41,19 +43,32 @@ int RequiredPluginClass = core::RegisterObject("Load the required plugins")
 
 RequiredPlugin::RequiredPlugin()
     : d_pluginName( initData(&d_pluginName, "pluginName", "plugin name (or several names if you need to load different plugins or a plugin with several alternate names)"))
+    , d_searchPath( initData(&d_searchPath, "searchPath", "Directory to scan before the default ones") )
     , d_suffixMap ( initData(&d_suffixMap , "suffixMap", "standard->custom suffixes pairs (to be used if the plugin is compiled outside of Sofa with a non standard way of differenciating versions), using ! to represent empty suffix"))
     , d_stopAfterFirstNameFound( initData(&d_stopAfterFirstNameFound , false, "stopAfterFirstNameFound", "Stop after the first plugin name that is loaded successfully"))
     , d_stopAfterFirstSuffixFound( initData(&d_stopAfterFirstSuffixFound , true, "stopAfterFirstSuffixFound", "For each plugin name, stop after the first suffix that is loaded successfully"))
     , d_requireOne ( initData(&d_requireOne , false, "requireOne", "Display an error message if no plugin names were successfully loaded"))
     , d_requireAll ( initData(&d_requireAll , true, "requireAll", "Display an error message if any plugin names failed to be loaded"))
 {
     this->f_printLog.setValue(true); // print log by default, to identify which pluging is responsible in case of a crash during loading
+    d_pluginName.setRequired(true);
```

Comment:
The name is not required, the altenrative is to set he pluginName in the name field which allows to write:
<RequiredPlugin name="Toto"/> instead of <RequiredPlugin pluginName="Toto"/>
The nice thing by using <RequiredPlugin name="Toto"/> is that it display in the scene graph view of runSofa the name of the plugin.

Reply:
I reverted this change as I understood the interest of having the same value for both `name` and `pluginName`: console output and GUI are more user-friendly this way.